com.onesignal.unity.core/Runtime/Notifications/Models/INotificationClickedResult.cs

```diff
@@ -32,15 +32,15 @@ namespace OneSignalSDK.Notifications.Models {
     /// See full documentation at
     /// https://documentation.onesignal.com/docs/sdk-notification-event-handlers#notification-opened-event
     /// </summary>
-    [Serializable] public sealed class NotificationClickedResult {
+    public interface INotificationClickedResult {

```

Comment:
The latest spec has this as an event, the result is contained in the event instead.